corehq/motech/openmrs/tests/test_repeaters.py

```diff
@@ -725,17 +725,29 @@ def test_delete_case_block_update_key(self):
         """)
         self.assert_expected_case_block(case_property, case_block_re)
 
-    def test_too_much_rope_error(self):
+    def test_deletable_kwargs_1(self):
         case_property = "case_id"
-        with self.assertRaises(TypeError):
-            delete_case_property(DOMAIN, "CASE_ID", case_property)
+        case_block_re = strip_xml(f"""
+            <case case_id="CASE_ID" »
+                  date_modified="{DATETIME_PATTERN}" »
+                  xmlns="http://commcarehq.org/case/transaction/v2">
+              <update>
+                <case_id />
```

Comment:
I was curious what this would do, so I submitted a case block like this locally.

Case History > Case Changes shows this:
![image](https://github.com/user-attachments/assets/b001626d-3531-49f3-8977-b11ae653d636)

But it does not create a new case property. The change is ignored.